### ISSUE TITLE
Dropdown Requires an extra mouse click to open.

### ISSUE THREAD

**Issue by WayneHiller** [Megabit/Blazorise#1748]
Blazorise.Bootstrap v.0.9.2.5

All the Dropdown (Buttons) on my pages require two clicks to open the dropdown list initially. Once opened they work correctly from then on until a page is revisited.

```
                    <Dropdown>
                        <DropdownToggle Color="Color.Primary">
                            Create
                        </DropdownToggle>
                        <DropdownMenu>
                            <DropdownItem>Request</DropdownItem>
                            <DropdownItem>Direct</DropdownItem>
                            <DropdownItem>Chat</DropdownItem>
                            <DropdownItem>Advertise</DropdownItem>
                        </DropdownMenu>
                    </Dropdown>
```

The very first click on a Dropdown Toggle focused the Button but does not open it. After this initial click the Dropdown works correctly with a single click.

**Comment by stsrki** [comment]
Sounds like something is interfering with dropdown attributes or styles. Can you check that you don't have duplicate imports of `bootstrap.css` or `bootstrap.js` files? Or even better, show us the entire content of your `index.html` or `_Host.cshtml`.

**Comment by stsrki** [comment]
Hi, any feedback on this? Did you check your files for duplicates?

**Comment by WayneHiller** [comment]
I am not seeing any bootstrap duplicate css. I am using SCSS with bootwatch as well. Plus Talerik UI for Blazor. so ya there is a lot going on. It is strange because it only seems to happen once in a while. Quite often now when I run the site it is fine from the start. I am still trying things to see if something if causing it to happen at runtime.  Not sure what I should look for that would cause this behavior.

**Comment by stsrki** [comment]
Maybe some javascript code is looking for `dropdown`, `dropdown-toggle`. Or maybe on `data-*` attribute, eg. `data-toggle`.

**Comment by WayneHiller** [comment]
@stsrki  Ya my thinking as well, seems like it is a timing issue or I might have to click something on the page to get it to start happening. Sometimes it seems like when the issue happens it is right from the start of the page but still trying to confirm that. I don't see how css would cause this issue, feels like it would have to be JS/C#

**Comment by WayneHiller** [comment]
What is also strange is that it only happens once to each of the dropdowns on the page. The first click on each sets the focus only without opening the list. But after that first click each one works perfectly after that. Every dropdown needs that first click to start working correctly. Like the initial "IsOpen" value is set wrong..

**Comment by stsrki** [comment]
Try commenting-out each `.js` file unti it starts working.

**Comment by WayneHiller** [comment]
@stsrki The problem is I can't even get it to misbehave now, the damn thing is working perfectly every time I run it lol. I will keep an eye on it and see what I can find the next time she decided to break.

**Comment by stsrki** [comment]
We had a similar problem that was recently fixed. Should be available in 0.9.5 in two weeks from now.
